validator/testcases/markup/markuptester.py

```diff
@@ -119,18 +116,15 @@ def _feed_parser(self, line):
         line = unicodehelper.decode(line)
 
         try:
-            self.feed(line + "\n")
-        except UnicodeDecodeError, exc_instance:
-            exc_class, val, traceback = sys.exc_info()
             try:
+                self.feed(line + "\n")
```

Comment:
Yeah, we do. This one is just to catch an encoding error and try falling back to ascii if it fails. The outer one is to catch an actual parser error. Arguably, this should just use `unicodehelper.decode`, but I'd rather not make that kind of change in this PR.